de-lib/src/main/java/org/iplantc/de/apps/client/presenter/tools/ManageToolsViewPresenter.java

```diff
@@ -23,32 +24,41 @@
     ManageToolsView.ManageToolsViewAppearance appearance;
 
     ToolServices dcService = ServicesInjector.INSTANCE.getDeployedComponentServices();
-    
+
     @Inject
     public ManageToolsViewPresenter(ManageToolsView.ManageToolsViewAppearance appearance) {
         this.appearance = appearance;
     }
 
     @Override
     public void go(HasOneWidget container) {
-       container.setWidget(toolsView.asWidget());
-       toolsView.mask(appearance.mask());
-       loadTools();
+        container.setWidget(toolsView.asWidget());
```

Comment:
I'm pretty sure you mentioned you're writing tests last, so I'll hold off on those comments 😜